### PR TITLE
Quick fixes for a couple of blockers

### LINKED ISSUE #3245: accessing exercise from progress report leads to incorrect sidebar behavior
screen shot 2015-03-10 at 3 03 52 pm
reproduce:
1.login as learner;
2.go to progress report(make sure you have something to choose)
3.open any drop down list and choose an exercise from it
you will see the sidebar fade are isn't fading, and the sidebar-tab is in 'close' state even though the sidebar is open

### PR DESCRIPTION
**Pull request by Antrikshy**
This is a quick middle-of-finals-week PR that I was supposed to open last weekend after a little bit of cleanup. While it's been some time since I worked on this, everything pushed into this branch right now should be stable.

This fixes #3245 and #3223.

### CODE REVIEW

File: kalite/distributed/static/css/distributed/sidebar.css [hunk 1]
```diff
@@ -141,6 +141,10 @@ div.fade {
     z-index: 10000;
 }
 
+.sidebar-back-hover {
+    background-color: #3D6663 !important;
```
Comment (MCGallaspy) [inline kalite/distributed/static/css/distributed/sidebar.css]: I can't distinguish this color from the non-hovered version. Maybe it's me?

**Comment by rtibbles** [comment]
@Antrikshy I think just wrapping the back button in a bootstrap button might be the simplest way to make it look clickable.

File: kalite/distributed/static/css/distributed/sidebar.css (hunk 1 above)
Comment (MCGallaspy) [inline kalite/distributed/static/css/distributed/sidebar.css]: #5AA685 seems better to me.

File: kalite/distributed/static/js/distributed/topics/views.js [hunk 2]
```diff
@@ -195,7 +195,7 @@ window.SidebarView = BaseView.extend({
         return false;
     },
 
-    update_sidebar_visibility: function() {
+    update_sidebar_visibility: _.debounce(function() {
```
Comment (MCGallaspy) [inline kalite/distributed/static/js/distributed/topics/views.js]: You know, I'm okay with this change.

**Comment by MCGallaspy** [comment]
Superseded by #3353.

**Comment by jamalex** [comment]
Thanks, @Antrikshy! Good luck with finals!

**Pull request closed without merging**